integration-test-common/src/test/java/org/apache/gravitino/integration/test/container/ContainerSuite.java

```diff
@@ -509,6 +538,10 @@ public HiveContainer getKerberosHiveContainer() {
     return kerberosHiveContainer;
   }
 
+  public HiveContainer getJdbcHiveContainer() {
```

Comment:
I think it better change `getJdbcHiveContainer()` to `getSqlBaseAuthHiveContainer()`?

Reply:
Renamed.